pages/api/graphql-rest.page.ts

```diff
@@ -12,6 +12,26 @@ import { Resolvers } from './graphql-rest.page.generated';
 const typeDefs = gql`
   type Query {
     contactFilters(accountListId: ID!): [ContactFilter!]!
+    taskAnalytics(accountListId: ID!): TaskAnalytics!
+  }
+
+  type TaskAnalytics {
+    id: ID!
+    type: String!
+    createdAt: ISO8601DateTime!
+    lastElectronicNewsletterCompletedAt: ISO8601DateTime
+    lastPhysicalNewsletterCompletedAt: ISO8601DateTime
+    tasksOverdueOrDueTodayCounts: [OverdueOrDueTodayTaskAnalytic!]!
+    totalTasksDueCount: Int!
+    updatedAt: ISO8601DateTime!
+    updatedInDbAt: ISO8601DateTime!
+  }
+
+  scalar ISO8601DateTime
```

Comment:
Was super happy to figure this out. I created a custom scalar here and then use it up in my `TaskAnalytics` type. On codegen, https://github.com/CruGlobal/mpdx-react/blob/7554b164b815d4fc08d60695e13d2c7cdea68524/codegen-server.yml#L17-L18 sees this scalar and will generate the correct type for it.